ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-cron/gio-form-cron.stories.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, moduleMetadata } from '@storybook/angular';
+import { Story } from '@storybook/angular/dist/ts3.9/client/preview/types-7-0';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormControl, FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+import { GioFormCronComponent } from './gio-form-cron.component';
+import { GioFormCronModule } from './gio-form-cron.module';
+
+export default {
+  title: 'Components / Form Cron',
+  component: GioFormCronComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [BrowserAnimationsModule, GioFormCronModule, FormsModule, ReactiveFormsModule, MatFormFieldModule],
+    }),
+  ],
+  argTypes: {
+    disabled: {
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    initialValue: {
+      control: { type: 'text' },
+    },
+  },
+  render: p => {
+    const control = new FormControl({ value: p.initialValue ?? '', disabled: p.disabled });
+
+    return {
+      template: `
+        <gio-form-cron [formControl]="control"></gio-form-cron>
+        <br>
+        --------------------
+        <br>
+        <input type="text" [formControl]="control" />
```

Comment:
Can we add a small text to explain what it is ? 😅  ( I'm asking because the input is editable so we can think that changing the value here should change the  gio-form-cron )

Reply:
I can remove this now. But in next Pr I've improve this part with value / touched / error display

Reply:
you can let it if you change it in next PR no problem 👍🏼